docs/src/Components.jsx

```diff
@@ -212,11 +212,11 @@ var Components = React.createClass({
 
           <article id="basket-item">
             <h3>Basket Item</h3>
-            <p>You can use the Basket Item as a way of representing anything with a price next to it. This means you could have a simple description (line of text) or even more complex markup passed in as the child.</p>
+            <p>You can use the Basket Item as a way of representing anything in a basket with or without a price next to it. This means you could have a simple description (line of text) or even more complex markup passed in as the child. Note to have a price, you must declare a title for the product. If the product has no title then description is used.</p>
```

Comment:
So originally something in a basket would always have a name and price. Hoever we looked at existing basket Items in out HX brand and sometimes something in a basket needs to be laid out like a basket item but is not a sellable product, therefore has no price.
E.G. Transfers...
<img width="247" alt="screen shot 2015-11-24 at 15 59 36" src="https://cloud.githubusercontent.com/assets/778942/11372284/6b005f92-92c4-11e5-839f-678d78cb1292.png">

We also deduced that an item can't have a price without a title. It just wouldn't make sense to the user.